test/parallel/test-dgram-bind-shared-ports.js

```diff
@@ -64,12 +64,15 @@ if (cluster.isMaster) {
 
   socket1.bind({
     address: 'localhost',
-    port: common.PORT,
+    port: 0,
     exclusive: false
-  }, () => {
-    socket2.bind({ port: common.PORT + 1, exclusive: true }, () => {
+  }, common.mustCall(() => {
+    socket2.bind({
+      port: socket1.address().port + 1,
```

Comment:
Yeah, both workers need to use the same two port numbers for both sockets. (In other words, each worker has two sockets. And those sockets should have duplicate ports across workers.)

I think the choices are:

* introduce code to communicate ports between the two workers, or
* move the test as-is to `sequential`